### DISCUSSION TITLE
Original clip name inside a synchronized

### DISCUSSION THREAD

**Discussion by HelgiThor161** [latenitefilms/Metaburner#2] [Ideas]
It would be really cool if it could read the original clip name inside a synchronized clip. 
I usually rename the sync clips to “scene-take” but the final burnin has to be original clip name. So I have to break apart the clips, copy&paste attributes, which is quite tedious.

**Comment by latenitefilms** [comment]
Thank you! I'll definitely try and make this happen.

**Reply by latenitefilms** [reply]
For anyone following along, this was added a while back.
